gobblin-core-base/src/main/java/gobblin/converter/AnyToSerializedRecordConverter.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package gobblin.converter;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+
+import gobblin.configuration.WorkUnitState;
+import gobblin.type.RecordWithMetadata;
+import gobblin.type.SerializedRecord;
+import gobblin.type.SerializedRecordWithMetadata;
+
+
+/**
+ * Converts multiple source objects to a SerializedRecord object.
+ */
+public class AnyToSerializedRecordConverter extends Converter<String, String, Object, SerializedRecord> {
+
+  private final static List<String> BINARY_CONTENT_TYPE = ImmutableList.of("application/octet-stream");
+  private final static List<String> JSON_CONTENT_TYPE = ImmutableList.of("application/json");
+  private final static Gson GSON = new GsonBuilder().disableHtmlEscaping().create();
```

Comment:
Any specific reason for turning off HTML escaping? 

We have different ways of creating new Gson objects in the codebase... should all of them disable HTML escaping?